audio_processing.py

```diff
@@ -24,29 +24,31 @@
 import tempfile
 import time
 import traceback
+import librosa
+import mimetypes
 from pprint import pformat
 from pathlib import Path
 
 import processing
 
 SLEEP_SECS = 10
+MAX_AMPLIFICATION = 20
 
 processing.init_logging()
 conf = processing.Config.load()
 
-# These input MIME types will be converted to mp4
-MIME_TYPES_TO_PROCESS = {
-    "video/3gpp": "3gpp",
-    "audio/3gpp": "3gpp",
-    "audio/wav": "wav",
-    "audio/x-flac": "flac",
-}
+mimetypes.add_type("audio/mp4", '.mp3')
+mimetypes.add_type("video/3gpp", '.3gpp')
+mimetypes.add_type("audio/3gpp", '.3gpp')
+mimetypes.add_type("audio/wav", '.wav')
+mimetypes.add_type("audio/x-flac", '.flac')
 
 BIT_RATE = "128k"
 
 
 def process(recording, api, s3):
-    input_extension = MIME_TYPES_TO_PROCESS.get(recording["rawMimeType"])
+    input_extension = mimetypes.guess_extension(recording["rawMimeType"])
+    newMetadata = {}
 
     if not input_extension:
```

Comment:
The comment and print statement need to be updated if `input_extension` is None. It's not that no processing is required, and more that the input format is unsupported.